### PR TITLE
New Components - interseller

### LINKED ISSUE #3659: Interseller
# interseller
## URLs
- https://interseller.readme.io/reference
#### Prompt
Alerts when a new contact is contacted. No props are required. Verify if the contact has the replied_at prop
- https://interseller.readme.io/reference/list-contacts
#### Prompt
Alerts when a new contact is created. No props are required
#### URLs
- https://interseller.readme.io/reference/list-contacts
#### Prompt
Create a new customer. Please add all optional and required props.
#### URLs
- https://interseller.readme.io/reference/list-contacts
- https://interseller.readme.io/reference/update-contact
#### Prompt
Update an existing contact to indicate as contacted. Please add props contactId, booked and sentment.
#### URLs
- https://interseller.readme.io/reference/list-contacts
- https://interseller.readme.io/reference/set-contact-replied

### PR DESCRIPTION
**Pull request by luancazarine**
Resolves #3659.

### CODE REVIEW

File: components/interseller/actions/set-contact-replied/set-contact-replied.mjs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    },
+    sentiment: {
+      propDefinition: [
+        interseller,
+        "sentiment",
+      ],
+    },
+  },
+  async run({ $ }) {
+    const response = await this.interseller.setContactReplied({
```
Comment (GTFalcao) [inline components/interseller/actions/set-contact-replied/set-contact-replied.mjs:31]: `$` should be passed here

File: components/interseller/actions/update-contact/update-contact.mjs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  props: {
+    interseller,
+    contactId: interseller.propDefinitions.contactId,
+    name: interseller.propDefinitions.name,
+    company: interseller.propDefinitions.company,
+    profileUrl: interseller.propDefinitions.profileUrl,
+    sourceUrl: interseller.propDefinitions.sourceUrl,
+    location: interseller.propDefinitions.location,
+    phoneNumber: interseller.propDefinitions.phoneNumber,
+    title: interseller.propDefinitions.title,
```
Comment (GTFalcao) [inline components/interseller/actions/update-contact/update-contact.mjs:18]: Even though this works by copying the attributes directly within the code, I think we should adjust to follow the standard propDefinition syntax

**Review by GTFalcao — COMMENTED** [review]
Hey @luancazarine there's just a couple of adjustments I recommend here.

File: components/interseller/sources/common/base.mjs [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      return this.db.get("lastDate") || 0;
+    },
+    _setLastDate(createdAt) {
+      this.db.set("lastDate", createdAt);
+    },
+    generateMeta(event, dateField) {
+      return {
+        id: event.id,
+        summary: this.getSummary(event),
+        ts: Date.parse(dateField),
```
Comment (michelle0927) [inline components/interseller/sources/common/base.mjs:26]:
```suggestion
        ts: Date.parse(event[dateField]),
```

**Review by michelle0927 — COMMENTED** [review]
Looks good, just one comment!

**Review by lcaresia — APPROVED** [review]

**Pull request merged**